src/main/java/it/pagopa/ecommerce/payment/methods/infrastructure/NpgSessionDocument.java

```diff
@@ -4,11 +4,16 @@
 import org.springframework.data.annotation.PersistenceConstructor;
 import org.springframework.data.redis.core.RedisHash;
 import org.springframework.lang.NonNull;
+import org.springframework.lang.Nullable;
 
 @RedisHash(value = "keys", timeToLive = 10 * 60)
 public record NpgSessionDocument(
         @NonNull @Id String sessionId,
-        @NonNull String securityToken
+        @NonNull String securityToken,
+        @Nullable String bin,
+        @Nullable String lastFourDigits,
+        @Nullable String expiringDate,
+        @Nullable String circuit
```

Comment:
If possible I would make a separate structure and declare that as nullable, i.e.:
```java
public record CardData(
    String bin,
    String lastFourDigits,
    String expiringDate,
    String circuit
) {}
```

```suggestion
        @Nullable CardData cardData
```

Reply:
Fixed with 3bfc15f394be9c0352023039ae79845dfe119d29!